mojave/items/clothing/head/helmet.dm

```diff
@@ -329,29 +341,35 @@
 /obj/item/clothing/head/helmet/ms13/ncr
 	name = "NCR infantry helmet"
 	desc = "A standard issue NCR infantry helmet adapted for arctic environments."
-	icon_state = "ncr_infantry_helmet_w"
-	inhand_icon_state = "ncr_infantry_helmet_w"
+	icon_state = "ncr_infantry_helmet"
+	inhand_icon_state = "ncr_infantry_helmet"
 	armor = list(melee = 35, bullet = 35, laser = 10, energy = 10, bomb = 20, bio = 10, rad = 10, fire = 10, acid = 10)
 	flags_cover = NONE
+	dynamic_hair_suffix = ""
+	dynamic_fhair_suffix = ""
 
 /obj/item/clothing/head/helmet/ms13/ncr/goggles
 	name = "NCR goggles helmet"
 	desc = "An NCR helmet with added goggles and additional protective padding."
-	icon_state = "ncr_goggles_helmet_w"
-	inhand_icon_state = "ncr_goggles_helmet_w"
+	icon_state = "ncr_goggles_helmet"
+	inhand_icon_state = "ncr_goggles_helmet"
 	can_toggle = 1
-	toggle_message = "You pull the goggles down on"
-	alt_toggle_message = "You push the goggles up on"
+	toggle_message = "You pull the goggles down"
```

Comment:
I'm fairly certain this toggle message has "on" because the full line is "You pull the goggles down on the NCR goggles helmet" and "You push the goggles up on the NCR goggles helmet". I'm not 100% sure this is the case, so probably worth testing to make sure. I just get a feeling the "on" part was there for a reason.

Reply:
Confirmed and fixed.